lib/cesium_link_web/live/link_live/index.html.heex

```diff
@@ -15,30 +15,46 @@
 </.header>
 
 <.table id="links" rows={@streams.links} phx-hook="Sorting">
-  <:col :let={{_id, _link}}><.icon name="hero-bars-3 cursor-pointer ml-4" class="handle w-5 h-5" /></:col>
+  <:col :let={{_id, _link}}>
+    <% publish_future = _link.publish_at && DateTime.compare(_link.publish_at, DateTime.utc_now()) == :gt %>
+    <.icon name="hero-bars-3 cursor-pointer ml-4" class={"handle w-5 h-5 #{if publish_future, do: "opacity-50"}"} />
+  </:col>
   <:col :let={{_id, link}} label="Name">
-    <p class="font-semibold text-zinc-900"><%= link.name %></p>
+    <% publish_future = link.publish_at && DateTime.compare(link.publish_at, DateTime.utc_now()) == :gt %>
+    <p class={"font-semibold text-zinc-900 #{if publish_future, do: "opacity-50"}"}><%= link.name %></p>
   </:col>
   <:col :let={{_id, link}} label="Emoji">
-    <.emoji code={link.emoji} />
+    <% publish_future = link.publish_at && DateTime.compare(link.publish_at, DateTime.utc_now()) == :gt %>
+    <span class={if publish_future, do: "opacity-50"}>
```

Comment:
You should create a function in the LiveView of this page like:

```elixir
defp publish_in_future?(link) do
  not is_nil(link.publish_at) && DateTime.after?(link.publish_at, DateTime.utc_now())
end
````

Then you can reuse it in all places and stop doing logic in the middle of the template :bulb:

Reply:
Or even better, in the Link resource, create a function:

```elixir
def published?(%Link{publish_at: nil}), do: true
def published?(%Link{publish_at: publish_at}), do: DateTime.after?(publish_at, DateTime.utc_now())
```

And them in the 

```elixir
defp publish_in_future?(link), do: not Link.published?(link)
```